### PR TITLE
#34 : RevokeWorkRequestController/Data access 

### PR DESCRIPTION
**Pull request by rameshthumati22**

### CODE REVIEW

**Comment by magwas** [comment]
please merge the develop branch of kode-konveyor/tidy-up and resolve merge conflicts first

**Comment by szirbucz** [comment]
Please resolve the merge conflicts

**Comment by magwas** [comment]
udated with develop. @rameshthumati22 do a git pull locally

File: sh.exe.stackdump [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Stack trace:
```
Comment (szirbucz) [inline sh.exe.stackdump]: Remove this file

File: src/test/java/com/kodekonveyor/work_request/revoke/RevokeWorkRequestControllerDataAccessTest.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+  @Test
+  @DisplayName("The country is returned corectly")
+  public void test6() {
+    assertEquals(
+        AddressEntityTestData.COUNTRY,
+        WorkRequestEntityTestData.get().getAddress().getCountry()
+    );
+  }
+
```
Comment (szirbucz) [inline src/test/java/com/kodekonveyor/work_request/revoke/RevokeWorkRequestControllerDataAccessTest.java]: You are testing the consistency of TestData classes, that makes no sense.
You should verify that WorkRequestRepository.findById with correct id.

File: src/test/java/com/kodekonveyor/work_request/revoke/RevokeWorkRequestControllerTest.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+  @Test
+  @DisplayName("The work request with the given id is deleted")
+  public void deleteWorkRequestById() {
+
+    revokeWorkRequestController
+        .call(WorkRequestEntityTestData.WORK_REQUEST_ID);
+
+    Mockito.verify(workRequestRepository)
+        .delete(WorkRequestEntityTestData.get());
```
Comment (szirbucz) [inline src/test/java/com/kodekonveyor/work_request/revoke/RevokeWorkRequestControllerTest.java]: It is not in the scope of your task.

**Comment by magwas** [comment]
Closing due to inactivity. If you would like to work on it further, just grab the task again and reopen it.

**Pull request closed without merging**